add-on/chrome/bootstrap.js

```diff
@@ -29,6 +29,9 @@ XPCOMUtils.defineLazyModuleGetter(this, "Task",
  * to provide the required loop functions for the window.
  */
 var WindowListener = {
+  // Records the add-on version once we know it.
+  addonVersion: "unknown",
```

Comment:
Why tack it to `WindowListener` and not as a global like you did in MozLoopService.jsm?